public/app/features/correlations/types.ts

```diff
@@ -8,6 +8,11 @@ export type GetCorrelationsResponse = Correlation[];
 
 type CorrelationConfigType = 'query';
 
+export enum TransformationType {
```

Comment:
Could we use `SupportedTransformationType` from `@grafana/data` instead?

Reply:
Ooops - yep! Fixed